text/java/0018-retire-java-azure-composite.md

```diff
@@ -0,0 +1,40 @@
+# Retire Java Azure Composite Buildpack
+
+## Summary
+
+This RFC is to retire and archive the Java Azure Composite buildpack.
+
+## Motivation
+
+1. The buildpack is not being maintained. It hasn't been updated in a long time.
+2. No one has complained that it's not been updated, which is a signal no one is using it.
+3. The buildpack is not ultimately necessary as the main feature is that it uses a different JVM, which one can do without creating an entirely new buildpack.
+4. We are introducing the jvm-vendors buildpack, which will even more easily allow users to pick different JVMs.
+5. We don't want to set an expectation that we will create composite buildpacks for every JVM vendor.
+
+## Detailed Explanation
+
+The process is simple.
+
+1. Write a blog post announcing the change. Provide instructions for how to continue using Microsoft JDK without this buildpack.
+2. Give users 30 days to update.
+3. Archive the buildpack on Github. This will stop all workflows.
```

Comment:
```suggestion
3. Archive the buildpack on Github. This will stop all workflows.
4. Delete the "latest" tag on the Docker Hub repository for `paketobuildpacks/java-azure`.
```